Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM golang:1.14-alpine as builder
+
+RUN apk update && apk add --no-cache git ca-certificates && update-ca-certificates
```

Comment:
can you explain in a comment why we need operations related to ca-certificates please ? 

it doesn't appear on the CLI v1, and it's not mentionned anywhere in the README of the v2